docs/reviewing.md

```diff
@@ -51,6 +53,10 @@ the order above resembles the execution order of methods on CI. therefore, for i
 The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instatiation of objects to ensure linkage
 and dependencies are correct.
 
+## License Attribute
+
+The manadatory license attribute of each recipe must **only** of [SPDX license](https://spdx.org/licenses/) [short Identifiers](https://spdx.dev/ids/)
```

Comment:
This does simply not work as there are tons of custom licenses out there that do not have an entry on SPDX. So it is more a "must, if possible" situation

Reply:
Perfect, that answer my opening questions!

Reply:
```suggestion
The mandatory license attribute of each recipe **should** be a [SPDX license](https://spdx.org/licenses/) [short Identifiers](https://spdx.dev/ids/) when applicable.
```